serve-docs.cmd

```diff
@@ -1 +0,0 @@
-PowerShell.exe -file "serve-docs.ps1" %*  .\docs\docfx.json --serve -p 8090
```

Comment:
Do we not need this anymore? How do we do a live preview of docs now?

Reply:
We do not need this any more. To do a live preview `build.cmd servedocs` with 8090 as default. The port can be changed this way `build.cmd servedocs --port 8989`. 
```csharp
 Target ServeDocs => _ => _
        .Description("Build and preview documentation")
        .Executes(() => DocFXServe(s=>s.SetFolder(DocFxDir).SetPort(Port)));

```